server.js

```diff
@@ -145,3 +150,38 @@ app.get('/come-back', (req, res) => {
 </body>
 </html>`);
 });
+
+const cspIds = new Map();
```

Comment:
Undeleted ids will eventually reset the server. Maybe not a huge deal, but maybe we can remove all entries older than 1m when adding new entry? Or have a hard limit of 20 entries and always override the oldest one?